pkg/apiserver/rest/dualwriter_mode3.go

```diff
@@ -57,7 +57,7 @@ func (d *DualWriterMode3) Create(ctx context.Context, obj runtime.Object, create
 	d.recordStorageDuration(false, mode3Str, d.resource, method, startStorage)
 
 	go func() {
-		ctx, cancel := context.WithTimeoutCause(ctx, time.Second*10, errors.New("legacy create timeout"))
+		ctx, cancel := context.WithTimeoutCause(context.Background(), time.Second*10, errors.New("legacy create timeout"))
```

Comment:
should things like the signed in user, request info, span, etc be copied to the new context?

Reply:
you're right, changing it :)